### PR TITLE
Fixed head camera placement / configurable robot cameras

### PR DESCRIPTION
**Pull request by ASzot**
## Motivation and Context

Fixed a bug with the placement of the head camera. 

Updated camera positions and added more flexibility to the camera definitions in the robot wrapper and cleaned up this code. 

## How Has This Been Tested

With the interactive play script from Habitat Lab. See output below. 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by ASzot** [comment]
https://user-images.githubusercontent.com/10453381/126020519-33997bdc-3243-4122-a795-a9a0c1685809.mov

File: habitat_sim/robots/mobile_manipulator.py [hunk 1]
```diff
@@ -111,15 +123,11 @@ def __init__(
...
-        ]
-        self._head_sensor_names = [
-            s for s in self._sim._sensors if s.startswith("robot_head_")
-        ]
-        self._third_sensor_names = [
-            s for s in self._sim._sensors if s.startswith("robot_third_")
-        ]
+        self._cameras = defaultdict(list)
+        for camera_prefix in self.params.cameras:
+            for s in self._sim._sensors:
```
Comment (mathfac) [inline habitat_sim/robots/mobile_manipulator.py:128]:
```suggestion
            for sensor in self._sim._sensors:
```

File: habitat_sim/robots/mobile_manipulator.py [hunk 2]
```diff
@@ -111,15 +123,11 @@ def __init__(
...
+                if s.startswith(camera_prefix):
```
Comment (mathfac) [inline habitat_sim/robots/mobile_manipulator.py:129]:
```suggestion
                if sensor.startswith(camera_prefix):
```

File: habitat_sim/robots/mobile_manipulator.py [hunk 3]
```diff
@@ -111,15 +123,11 @@ def __init__(
...
+                    self._cameras[camera_prefix].append(s)
```
Comment (mathfac) [inline habitat_sim/robots/mobile_manipulator.py:130]:
```suggestion
                    self._cameras[camera_prefix].append(sensor)
```

**Review by mathfac — APPROVED** [review]
Looks better now, some small comment.

**Comment by ASzot** [comment]
@mathfac updated based on comments, will merge once tests pass.

**Pull request merged**